tests/smoke/login/login-success.test.ts

```diff
@@ -0,0 +1,11 @@
+import { test, expect } from '@playwright/test';
+import { LoginPage } from '../../../page-objects/login-page';
+import dotenv from 'dotenv';
```

Comment:
You can add this rows off code for .env config into playwright.conf.ts file - then you won't need to duplicate them in  each test file